src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -9,6 +9,7 @@ import Gallery from "react-grid-gallery";
 import styles from "./attachments-overview.css";
 import CameraUpload from "./camera-upload.component";
 import { Trans } from "react-i18next";
+import { UserHasAccessReact } from "@openmrs/esm-api";
```

Comment:
@FlorianRappl @jonathandick I find this naming weird: `UserHasAccessReact`, why "react"?

Reply:
That is not my choice, but I guess the intention was to signal that the given component is actually a React component.